frontend/package.json

```diff
@@ -23,6 +23,7 @@
     "react-router-dom": "^6.18.0"
   },
   "devDependencies": {
+    "@actions/core": "^1.10.1",
```

Comment:
I don't think this package is needed (unless I miss something)?

Reply:
I have removed it now, 
it was for me to locally test the workflow but it is not needed as I see I can test directly on the branch in github